cmp/grid/Grid.js

```diff
@@ -407,6 +454,20 @@ export class Grid extends Component {
         }
     };
 
+    // Catches row group changes triggered from ag-grid ui components
+    onColumnRowGroupChanged = (ev) => {
+        if (ev.source !== 'api' && ev.source !== 'uiColumnDragged') {
```

Comment:
This event (as well as columnVisible) will get fired when the user is dragging the column around in the ag-grid panels, so we want to ignore if source is uiColumnDragged and let the onDragEnd handler update our model once the change has been finished